README.md

```diff
@@ -22,11 +22,20 @@ nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase
 ```
 
 ### Install extcap script
+
+#### Linux
 Copy the provided script to the appropriate directory:
 ```
 sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
```

Comment:
The correct destination for the extcap utility can be found in "About Wireshark" -> "Folders" -> "Extcap path" for each OS.